admin/acceptance_tests/__init__.py

```diff
@@ -122,3 +122,12 @@ def _check_select(self, select, expected):
                 assert exp['value'] == option['value']
             if 'selected' in exp:
                 assert exp['selected'] == ('selected' in option.attrs)
+
+    @staticmethod
+    def check_one_submission_problem(expected_msg, resp):
+        assert 'There was a problem with your submission' == \
+            resp.html.select_one('div[class="error-msg-lbl"]').text
+        assert 'Errors have been highlighted below' == \
+            resp.html.select_one('div[class="error-msg-detail"]').text
+        assert expected_msg == \
+            resp.html.select_one("[class~'has-error']").select_one("[class~'help-block']").getText()
```

Comment:
This is good, but for consistency, I would prefer keeping same prototype as others checks for the moment : not static and protected, ex: 
``def _check_submission_problem(elem, expected_msg):``
or
``def _check_submission_problem(resp, item, expected_msg):``

Note that other tests could be factorized, maybe later.